app/assets/stylesheets/_devise_error_messages.scss

```diff
@@ -0,0 +1,3 @@
+.message_width {
+  max-width: 500px;
+}
```

Comment:
If you port the devise messages to the dynamic notification like flash messages this will be deleted. Either way, this width constraint should be left to the Bootstrap's grid system.